src/app/components/templates/DefaultPages/Delivery.tsx

```diff
@@ -61,8 +61,8 @@ const DeliveryInfo: React.FC = ()=> {
     <InfoBody>
         <Information>
             <Highlight>Kasia terms and conditions</Highlight>
-            <Description>Please read these Terms and Conditions (&quot;Terms&quot;, &quot;Terms and Conditions&quot;) carefully before using the application or https://www.plusity.com website (the &quot;Service&quot;)
-             operated by PLUSITY (&quot;us&quot;, &quot;we&quot;, or &quot;our&quot;).</Description>
+            <Description>Please read these Terms and Conditions ("Terms", "Terms and Conditions") carefully before using the application or https://www.plusity.com website (the "Service")
+             operated by PLUSITY ("us", "we", or "our").</Description>
```

Comment:
I made changes to the way single quotes, double quotes and apostrophes are rendered. It wasn't getting compiled when deploying, so for this same since you are using here a double quotes, you need to  replace `" `with `&quot;`